fvm/transactionStorageLimiter.go

```diff
@@ -155,3 +163,14 @@ func (limiter TransactionStorageLimiter) checkStorageLimits(
 
 	return nil
 }
+
+// shouldSkipSpecialAddress returns true if the address is a special address where storage
+// limits are not enforced.
+// This is currently only the EVM storage address. This is a temporary solution.
+func (limiter TransactionStorageLimiter) shouldSkipSpecialAddress(
+	ctx Context,
+	address flow.Address,
+) bool {
+	sc := systemcontracts.SystemContractsForChain(ctx.Chain.ChainID())
```

Comment:
I wonder if we should use the `evm.RootAccountAddress()` to get the address, since now if the implementation there changes this might not work.